namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/pushgateway.tf

```diff
@@ -0,0 +1,6 @@
+module "pushgateway" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-pushgateway?ref=1.2"
```

Comment:
Can you bump this down to 1.1. 1.2 contains the necessary code for 0.13. Really sorry for the confusion here.